npy_array/WORKSPACE.bazel

```diff
@@ -0,0 +1,45 @@
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository", "new_git_repository")
```

Comment:
I don't have much experience with WORKSPACE.bazel and .bazelrc. I trust that this set-up is state of the art :)

Reply:
Yes they are :).